slate-video-library/src/Components/Video-Card/Videocard.js

```diff
@@ -1,24 +1,29 @@
 import React from "react";
+
 import "./Videocard.css";
-function Videocard() {
+function Videocard({ video }) {
+  const { _id, title, videoUrl, creator_pic } = video;
+  console.log(video);
   return (
     <div>
       <div className="video__cards">
         <iframe
           className="video__cards-iframe-videos"
           title=" videos"
           alt="videodata"
-          src="https://www.youtube.com/embed/tgbNymZ7vqY"
+          src={videoUrl}
         ></iframe>
         <div className="video__cards-details">
           <div className="video__cards-image">
-            <img src="/" alt="images" className="" /> channel title
+            <img src={creator_pic} alt="images" className="" />
```

Comment:
suggestion:alt should not contain text like picture,image,img etc ,instead you can write "creator"